4.1/crud-fields.md

```diff
@@ -1806,6 +1810,8 @@ Show a [Summernote wysiwyg editor](http://summernote.org/) to the user.
 
 ```
 
+> NOTE: Please take into consideration that the contents displayed in the editor are not escaped by default. You should take care about escaping the contents, or don't allow it to reach the database in the first place.
```

Comment:
```suggestion
> NOTE: Summernote does NOT sanitize the input. If you do not trust the users of this field, you should sanitize the input or output using something like HTML Purifier. Personally we like to use install [mewebstudio/Purifier](https://github.com/mewebstudio/Purifier) and add an [accessor or mutator](https://laravel.com/docs/8.x/eloquent-mutators#accessors-and-mutators) on the Model, so that wherever the model is created from (admin panel or app), the output will always be clean. [Example here](https://github.com/Laravel-Backpack/demo/commit/7342cffb418bb568b9e4ee279859685ddc0456c1).
```